sys/net/gnrc/application_layer/dhcpv6/Kconfig

```diff
@@ -0,0 +1,17 @@
+menuconfig KCONFIG_MODULE_GNRC_DHCPV6
+    bool "Configure GNRC-part of DHCPv6"
+    depends on MODULE_GNRC_DHCPV6
+    help
+        Configure GNRC-part of DHCPv6 via Kconfig.
+
+if KCONFIG_MODULE_GNRC_DHCPV6
+
+if MODULE_GNRC_DHCPV6_CLIENT_6LBR
+config GNRC_DHCPV6_CLIENT_6LBR_UPSTREAM
```

Comment:
This seems like it could be better modeled as a `bool`

Reply:
Sorry, misread the help. Nevermind

Reply:
Huh? The interface identifiers are numerical for GNRC.

Reply:
0 here is just "not defined" i.e. "take first interface you find", as documented.

Reply:
> Huh? The interface identifiers are numerical for GNRC.

Yeah, sorry for the noise. I misunderstood the option :-)